app/controllers/gobierto_admin/gobierto_people/people_controller.rb

```diff
@@ -6,7 +6,7 @@ class PeopleController < BaseController
       before_action { module_enabled!(current_site, "GobiertoPeople") }
 
       def index
-        @people = current_site.people.sorted
+        @people = current_site.people.includes(:events, :statements, :posts).sorted
```

Comment:
Hey, I ended up configuring a counter cache strategy for those associations to avoid messing with kind of raw queries. It looks a bit premature to me but I feel much better now. What do you think? Feel free to punch the merge button if that looks good to you!